compiler/src/parsing/lexer.mll

```diff
@@ -96,6 +108,8 @@ rule token = parse
   | comment { process_newlines lexbuf; token lexbuf }
   | blank { token lexbuf }
   | newline_chars { process_newlines lexbuf; EOL }
+  | (signed_int as x) 'l' { INT32 (Int32.of_string x) }
+  | (signed_int as x) 'L' { INT64 (Int64.of_string x) }
```

Comment:
Where's the inspiration for this syntax from?

Reply:
It's the same syntax as C/C++/Java/OCaml. JavaScript uses the `n` suffix for `BigInt` but that doesn't feel quite right since other languages use `n` for whatever the system's native int size is.

Some references:
https://docs.microsoft.com/en-us/cpp/c-language/c-floating-point-constants?view=vs-2019
https://en.wikiversity.org/wiki/Advanced_Java/Declaring_float_and_long_Literals

Reply:
Okay, cool. Looks good.